### DISCUSSION TITLE
lock -r option

### DISCUSSION THREAD

**Discussion by kgunja** [pypa/pipenv#5790] [Q&A] [answered]
Hi, 
I just been handed some older code to enhance, and the project is using pipenv. It's attempting to use 'pipenv lock -r' which is no longer an option, what was the -r option for?
Thanks
K

**Comment by matteius** [comment, marked as answer]
@kgunja it generates a requirements file which has been replaced by the `requirements` command to simplify the routines/code paths.

**Reply by kgunja** [reply]
ok thanks
